string frame_id
Circle[] circles